onedal/datatypes/utils/dtype_conversions.hpp

```diff
@@ -0,0 +1,53 @@
+/*******************************************************************************
+* Copyright 2023 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#pragma once
+
+#include <string>
+
+#include <pybind11/pybind11.h>
+
+#include "oneapi/dal/common.hpp"
+
+namespace py = pybind11;
+
+#define SET_DAL_TYPE_FROM_DAL_TYPE(_T, _FUNCT, _EXCEPTION) \
```

Comment:
The naming is misleading. Should it be `SET_SUA_TYPE_FROM_DAL_TYPE`?

Reply:
thanks. Changed.